### PR TITLE
Support optee guest base on grub loader on Kaby Lake

### PR DESCRIPTION
**Pull request by GangSecurity**
Signed-off-by: Chen Gang G <gang.g.chen@intel.com>

### CODE REVIEW

File: packer/Makefile [hunk 1]
```diff
@@ -6,8 +6,13 @@
 #
 ################################################
 
+ifneq (, $(findstring -DMODULE_OPTEE_GUEST, $(EVMM_CMPL_FLAGS)))
```
Comment (kwang13) [inline packer/Makefile:9]: this is not good. can we export CC = clang in optee.cfg, and check if CC has value here?

File: packer/Makefile [hunk 2]
```diff
@@ -58,7 +65,7 @@ ifneq (, $(findstring -DPACK_LK, $(EVMM_CMPL_FLAGS)))
 	if [ $(LKBIN_DIR) ]; then cp $(LKBIN_DIR)lk.bin $(BUILD_DIR); fi
 else
 ifneq (, $(findstring -DPACK_OPTEE, $(EVMM_CMPL_FLAGS)))
-	if [ $(OPTEEBIN_DIR) ]; then cp $(OPTEEBIN_DIR)tee-pager.bin $(BUILD_DIR); fi
+	if [ $(OPTEEBIN_DIR) ]; then cp $(OPTEEBIN_DIR)tee-pager_v2.bin $(BUILD_DIR); fi
```
Comment (kwang13) [inline packer/Makefile:68]: why v2? it is better not to add version in a name of a binary, especially there's only 1 binary.

Comment (GangSecurity) [reply packer/Makefile:68]: There is no tee-pager.bin in optee-3.13, it is replaced by tee-pager_v2.bin

File: packer/Makefile (hunk 1 above)
Comment (GangSecurity) [reply packer/Makefile:9]: can we use below?
ifneq (, $(CLANG_BINDIR))
export CC = clang
else 
export CC = gcc
endif

**Review by kwang13 — APPROVED** [review]

**Pull request merged**